BTDBTest/ObjectDbTableFreeContentTest.cs

```diff
@@ -746,6 +747,45 @@ static string DumpUnseenKeys(FindUnusedKeysVisitor visitor, string concat)
             return builder.ToString();
         }
 
+        [Fact]
+        public void PossibleToEnumerateRelations()
+        {
+            using (var tr = _db.StartTransaction())
+            {
+                tr.InitRelation<ILinks>("LinksRelation")(tr);
+                tr.InitRelation<ISetings>("SetingsRelation")(tr);
+                tr.InitRelation<IFileTable>("FileRelation")(tr);
+                CheckRelationTypes(tr.EnumerateRelationTypes());
+                tr.Commit();
+            }
+
+            using (var tr = _db.StartTransaction())
+            {
+                CheckRelationTypes(tr.EnumerateRelationTypes());
+            }
+
+            ReopenDb();
+
+            using (var tr = _db.StartTransaction())
```

Comment:
Code duplicity except for calling `Commit()` at the end. Just use nested method and call it twice?